uk.co.compucorp.civicrm.tasksassignments/js/test/tasks-assignments/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -67,6 +67,30 @@ define([
       });
     });
 
+    describe('workflow activity types', function () {
+      var expectedWorkflowActivityTypes;
+
+      beforeEach(function () {
+        var caseTypes = assignmentFabricator.assignmentTypes();
+        var CATEGORY_FIELD = 'custom_100';
+        var WORKFLOW_TYPE = 'Workflow';
+
+        $rootScope.cache.assignmentType.obj = caseTypes;
+        $rootScope.cache.assignmentType.arr = _.values(caseTypes);
+        expectedWorkflowActivityTypes = _.values(caseTypes).filter(function (caseType) {
+          return caseType[CATEGORY_FIELD] === WORKFLOW_TYPE;
+        });
+
+        initController({
+          config: { customFieldIds: { 'caseType.category': 100 } }
+        });
+      });
+
+      it('stores a list of workflow activity types', function () {
```

Comment:
Please change this to `stores a list of workflow activity types only`. This way we know that its supposed to return only the workflow types.